shell/common/pipeline.h

```diff
@@ -60,20 +60,22 @@ class Pipeline : public fml::RefCountedThreadSafe<Pipeline<R>> {
       }
     }
 
-    void Complete(ResourcePtr resource) {
+    bool Complete(ResourcePtr resource) {
```

Comment:
These seem like a good candidates to decorate with `FML_WARN_UNUSED_RESULT`.

Reply:
Will do!

Reply:
FML_WARN_UNUSED_RESULT doesn't seem to exist anymore. Removed in https://github.com/flutter/engine/commit/a37560a07447befc74e5c418f00cac0cad4e1404
Should we bring it back? Maybe in a seperate PR?

Reply:
Sorry, I meant [[nodiscard]] which is the C++17 way of specifying `FML_WARN_UNUSED_RESULT`.

Reply:
done!